features/torque2/client/job-features.pan

```diff
@@ -38,28 +22,82 @@ function torque_size_to_gb = {
     };
     if (match(ARGV[0], 'tb')) {
         size = split('tb', ARGV[0]);
-        return(to_long(size[0]) * 1000);
+        return(to_long(size[0]) * 1099511627776);
```

Comment:
Ah ok, well the base unit for almost all components that deal with storage is 1MB so that's why it's like that.